src/interpreters/mitie_sklearn_interpreter.py

```diff
@@ -1,12 +1,17 @@
-from mitie import named_entity_extractor, text_categorizer
+from mitie import named_entity_extractor
+import cloudpickle
 from rasa_nlu import Interpreter
+from rasa_nlu.featurizers.mitie_featurizer import MITIEFeaturizer
 from rasa_nlu.tokenizers.mitie_tokenizer import MITIETokenizer
 
 
 class MITIESklearnInterpreter(Interpreter):
-    def __init__(self, metadata):
-        self.extractor = named_entity_extractor(metadata["entity_extractor"])  # ,metadata["feature_extractor"])
-        self.classifier = text_categorizer(metadata["intent_classifier"])  # ,metadata["feature_extractor"])
+    def __init__(self, intent_classifier=None, entity_extractor=None, feature_extractor=None, **kwargs):
```

Comment:
naming (I know its the same in the `MITIEInterpreter`, but lets try to improve the code base 😅): all of these three names should indicate that they are strings representing a path not the actual classifier / extractor.